openapi/anchor-platform/versions/callbacks-2.9.0.yaml

```
@@ -0,0 +1,1309 @@
+openapi: 3.1.0
+info:
+  version: 3.0.0
```

```suggestion
  version: 2.9.0
```

When I was working on #909, I changed the `info.version` in the specfiles to `3.0.0`, 'cause I thought that would be next up. You'll probably want to change that in these files. Maybe even in the `/openapi/anchor-platform/main-*.yaml` files, too, depending on what your release cycles look like

---

Thanks for the catch! `2.9` should be the last minor version before our `3.0` release, so I think it makes sense to only update the versions in `/openapi/anchor-platform/*-2.9.0.yaml`s.

---

Is there a way to know what's changed from `2.8.x` to `2.9.0`?